relayer/crates/starknet-chain-components/src/types/messages/ibc/ibc_transfer.rs

```diff
@@ -29,12 +35,46 @@ pub type EncodeIbcTransferMessage = CombineEncoders<
     ],
 >;
 
+#[derive(HasField)]
+pub struct IbcTransferSendMessage {
+    pub port_id_on_a: PortId,
+    pub chan_id_on_a: ChannelId,
+    pub packet_data: IbcTransferMessage,
```

Comment:
What about renaming its type to something makes more sense? e.g. `IbcTransferPacketData`?

Reply:
I used `IbcTransferSendMessage` as it is a message that is submitted to the ics20 contract. But I can refactor the names more sensibly and consistent with Cairo counterparts.

Reply:
9881575